pkgs/by-name/ba/bash-language-server/package.nix

```
@@ -19,9 +20,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-yJ81oGd9aNsWQMLvDSgMVVH1//Mw/SVFYFIPsJTQYzE=";
   };
 
-  pnpmWorkspace = "bash-language-server";
+  pnpmWorkspaces = [ "bash-language-server" ];
   pnpmDeps = pnpm_8.fetchDeps {
-    inherit (finalAttrs) pname version src pnpmWorkspace;
+    inherit (finalAttrs)
+      pname
+      version
+      src
+      pnpmWorkspace
```

Same for `astro-language-server`
```suggestion
      pnpmWorkspaces
```

---

Fixed both of these instances.